grype/presenter/json/presenter_test.go

```diff
@@ -142,7 +141,7 @@ func TestJsonPresenter(t *testing.T) {
 		},
 	}
 
-	results := result.NewResult()
+	results := match.NewResult()
```

Comment:
shouldn't this now be `match.NewMatch()` ?

Reply:
great catch, I'll update